src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -221,6 +221,7 @@ protected String getPseudoElementValue(WebElement element, String pseudoElement,
 
   protected void scrollAndClick(WebElement element) {
     scrollToElement(element);
+    waitForElementClickableByElement(element);
```

Comment:
I'm wondering where and how the timetout change should be added then. 
Maybe we should  create method "changeWaitTimeout(int timeout)" that performs that and do this:

changeWaitTimeout(5);
waitForElementClickableByElement(element);
changeWaitTimeout(30);